modules/effects/spec/effect_sources.spec.ts

```diff
@@ -411,8 +328,9 @@ describe('EffectSources', () => {
       const d = { not: 'a valid action' };
       const e = undefined;
       const f = null;
-      const i = { type: 'From Source Identifier' };
-      const i2 = { type: 'From Source Identifier 2' };
+      const identifierAction1 = { type: 'From Source Identifier' };
```

Comment:
I didn't expect this file to become this messy.
I renamed the identifier actions and added describe blocks, I can revert these changes to make this PR readable.

Reply:
Yea, its kinda difficult to see what changes are related.

Reply:
@brandonroberts should be better now